app/assets/javascripts/discourse/app/components/edit-category-general.hbs

```diff
@@ -19,13 +19,13 @@
       <label>{{i18n "category.parent"}}</label>
       <CategoryChooser
         @value={{this.category.parent_category_id}}
-        @categories={{this.parentCategories}}
         @allowSubCategories={{true}}
         @allowRestrictedCategories={{true}}
         @onChange={{action (mut this.category.parent_category_id)}}
         @options={{hash
           allowUncategorized=false
           excludeCategoryId=this.category.id
+          autoInsertNoneItem=true
```

Comment:
There's a possible system test for this: visit category edit page and ensure that select-kit has the `none` option. Maybe could test multiple setting configs as well (with uncategoried enabled, without uncategorized enabled).

Reply:
I added an acceptance QUnit test for the category edit page. I preferred QUnit tests because we already have a bunch of use cases there and testing select kit is simpler (due to helpers).